src/Select.js

```diff
@@ -1703,7 +1696,7 @@ export default class Select extends Component<Props, State> {
               onTopArrive={onMenuScrollToTop}
               onBottomArrive={onMenuScrollToBottom}
             >
-              <ScrollBlock emotion={this.emotion} isEnabled={menuShouldBlockScroll}>
```

Comment:
we don't want to be doing this anymore, it locks us into emotion as an intrinsic  part of our components API, while this is true of our own implementations, this should ultimately not be true for power users.